src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Stores/OpenIdAuthorizationStore.cs

```diff
@@ -295,11 +295,10 @@ public virtual async ValueTask<long> PruneAsync(DateTimeOffset threshold, Cancel
 
             var authorizations = (await _session.Query<TAuthorization, OpenIdAuthorizationIndex>(
                 authorization => authorization.CreationDate < threshold.UtcDateTime &&
-                                (authorization.Status != OpenIddictConstants.Statuses.Valid ||
-                                (authorization.Type == OpenIddictConstants.AuthorizationTypes.AdHoc &&
+                                (authorization.Status != Statuses.Valid || authorization.Type == AuthorizationTypes.AdHoc) &&
```

Comment:
Note: that behavior change is deliberate: https://github.com/openiddict/openiddict-core/pull/2251.